### PR TITLE
Review-trials

### PR DESCRIPTION
**Pull request by piesieomane**
This is a project done by Vitalis and Piesie. It was a project set to test how we will collaborate as developers

### CODE REVIEW

**Review by piesieomane — COMMENTED** [review]
# Changes Required ♻️
### Hi @piesieomane, and @Vitalis

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- Your design matched that of the project. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] ## Required Changes :recycle:
_Check the comments under the review._
- Please make sure to add linters files
- Please add .gitignore file
- Please add a more descriptive Readme file.
- Please check your <.github/workflows> file
- Please add a more descriptive PR summary

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
```
Comment (piesieomane) [inline app.css:1]: - [ ] Please do not add blank lines, spaces, or indentations without a reason.

File: app.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+@media only screen and (max-width:400px) {
+    .main_img{
+        width: 400px;
+    }
+}
+.main_figure>figcaption{
+    padding: 20px 0px 20px;
+}
+
+
```
Comment (piesieomane) [inline app.css:47]: - [ ] Please do not add blank lines, spaces, or indentations without a reason.
  [Hint] Kindly delete one blank line

File: app.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                    scientific vision can make to human peace and progress."
+                  </p>
+                  <p>-- Indian Prime Minister Manmohan Singh</p>
+            </div>
+            <h3>
+                            
+                If you have time, you should read more about this incredible human being
+                on his
+                <a  class="mylink" href="http://https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank" >Wikipedia entry.</a>                
+            </h3>
```
Comment (piesieomane) [inline app.html:142]: - [ ] Kindly  replace `<h3>` with `<h2>`.  Kindly note this: Do not skip heading levels. Use the `<h1>` as the main headings of the webpage, followed by the `<h2>` headings, then the less important `<h3>` headings, etc.

**Review by m05e5 — APPROVED** [review]
# Hi @piesieomane , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**